src/index.html

```diff
@@ -9,6 +9,20 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header" id="home">
+      <a href="#home"><img src="./images/logo.png" alt="logothype"></a>
+      <nav class="nav_list">
+        <ul class="links">
+          <li><a href="index.html" class="link is-active">Apple</a></li>
```

Comment:
```suggestion
          <li><a href="#apple" class="link is-active">Apple</a></li>
```
change href to all a tags